### PR TITLE
Convert voltage to float

### PR DESCRIPTION
**Pull request by JAAlvarado-Montes**
This small PR fixes a bug where voltage if being left as string and not converted to float.

### CODE REVIEW

File: src/huntsman/pocs/dome/musca.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def door_closed(self):
     @property
     def is_safe_to_open(self):
         v = self.status[Protocol.BATTERY]
-        if v < self.MIN_OPERATING_VOLTAGE:
+        if float(v) < self.MIN_OPERATING_VOLTAGE:
```
Comment (AnthonyHorton) [inline src/huntsman/pocs/dome/musca.py]: This is the wrong place for this change. The function that gets the status for the dome should be converting number string to floats before it returns the status dict.

**Review by AnthonyHorton — CHANGES_REQUESTED** [review]

File: src/huntsman/pocs/dome/musca.py (hunk 1 above)
Comment (AnthonyHorton) [reply src/huntsman/pocs/dome/musca.py]: In this function, should convert `status[Protocol.BATTERY]` and probably `status[Protocol.SOLAR_ARRAY]` to floats before returning the `status` dict :

https://github.com/AstroHuntsman/huntsman-pocs/blob/4949428c09eb27f08c19f516a35203e5dc7db711/src/huntsman/pocs/dome/musca.py#L122

**Comment by JAAlvarado-Montes** [comment]
Is that better @AnthonyHorton ? Just addressed your comments :)

**Review by AnthonyHorton — APPROVED** [review]
Looks good to me. Hopefully there is no code elsewhere that expects these to be strings....

**Pull request merged**